### DISCUSSION TITLE
Resize image

### DISCUSSION THREAD

**Discussion by wannesmatthys** [awcodes/scribble#60] [General]
Hi,

In version v0.0.4-alpha, I could resize a Media.
![image](https://github.com/user-attachments/assets/108f7431-bd11-4de7-95bf-cf735499dfe6)

Yet, I can't seem to do this v0.0.7-alpha. Was this removed or do I need to enable a certain setting?
![image](https://github.com/user-attachments/assets/f90c60e1-0e9f-4f27-8aac-1db06b0693e2)

Thanks!

**Comment by awcodes** [comment]
Yea. It was removed for now until I can figure out how to make it work and still allow the image to be aligned. Sorry.

**Reply by wannesmatthys** [reply]
Alright, no worries. Thank you!
